plugins/inputs/sqlserver/sqlserver_test.go

```diff
@@ -138,13 +139,59 @@ func TestSqlServer_MultipleInstanceIntegration(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, s.isInitialized, true)
 	assert.Equal(t, s2.isInitialized, true)
-	// acc includes size metrics, and excludes memory metrics
+
+	// acc includes size metrics, and excludes memory metrics and the health metric
+	assert.False(t, acc.HasMeasurement(healthMetricName))
 	assert.False(t, acc.HasMeasurement("Memory breakdown (%)"))
 	assert.True(t, acc.HasMeasurement("Log size (bytes)"))
 
-	// acc2 includes memory metrics, and excludes size metrics
+	// acc2 includes memory metrics and the health metric, and excludes size metrics
+	assert.True(t, acc2.HasMeasurement(healthMetricName))
 	assert.True(t, acc2.HasMeasurement("Memory breakdown (%)"))
 	assert.False(t, acc2.HasMeasurement("Log size (bytes)"))
+
+	sqlInstance, database := getConnectionIdentifiers(testServer)
+	tags := map[string]string{healthMetricInstanceTag: sqlInstance, healthMetricDatabaseTag: database}
+	assert.True(t, acc2.HasPoint(healthMetricName, tags, healthMetricAttemptedQueries, 9))
+	assert.True(t, acc2.HasPoint(healthMetricName, tags, healthMetricSuccessfulQueries, 0))
```

Comment:
Please don't change this test-case! Here we have a nice indicator if the old behavior still exists with the previous settings. You can easily duplicate this test with the changes you propose but leave the old one unchanged!

Reply:
Good point.

Reply:
Please close as resolved.